### PR TITLE
Added 40KB message size test case

### LINKED ISSUE #451: 40KB message producer consumer test case
We can do this by extending @hasithajayasundara work on test suit #446

### PR DESCRIPTION
**Pull request by hasithajayasundara**
resolve #451

### CODE REVIEW

**Comment by a5anka** [comment]
Shall we rebase this with the master?

File: tools/scripts/run_broker_publisher.sh [hunk 1]
```diff
@@ -200,14 +200,17 @@ case ${testplan_user_inputs["MessageSize"]} in
...
+		;;
 	100KB)
         text_message_file_location="sample_messages/100kB.json"
 		;;
 	1MB)
 		text_message_file_location="sample_messages/1MB.json"
 		;;
 	?)
-	    echo message_size parameter should be one of 1KB,10KB,100KB and 1MB
+	    echo message_size parameter should be one of 1KB,10KB,40KB,100KB and 1MB
```
Comment (sdkottegoda) [inline tools/scripts/run_broker_publisher.sh]: There should be a space after each comma

**Comment by sdkottegoda** [comment]
Shall we squash the commits into one?

File: tools/scripts/run_broker_publisher.sh (hunk 1 above)
Comment (hasithajayasundara) [reply tools/scripts/run_broker_publisher.sh]: Done

**Pull request merged**